packages/google_sign_in/google_sign_in_web/example/integration_test/src/create_jwt.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:google_identity_services_web/id.dart';
+import 'package:jose/jose.dart';
+
+import 'jsify_as.dart';
+
+/// Wraps a key-value map of [claims] in a JWT token similar GIS'.
+///
+/// Note that the encryption of this token is weak, and this method should
+/// only be used for tests!
```

Comment:
I'm not going to add `visibleForTesting` here, because the above is just a friendly reminder. If you end up digging this code, it comes from an auxiliary function for an integration test on an example app... very far from other actual code... I think one who wanted to copy-paste this, would just remove the `@visibleForTesting` annotation anyway.